.agent-versions.json

```
@@ -1,10 +1,10 @@
 {
   "testVersions": [
-    "8.13.0-SNAPSHOT",
+    "8.13.2-SNAPSHOT",
+    "8.13.1",
+    "8.13.0",
```

```suggestion
    "8.13.1",
    "8.12.2",
```

No point testing all the 8.13.x patches. Except this, this looks good. Will manually edit and we can follow up to automate this selection.

---

@cmacknz So, looks like we want 2 previous minors then. The current version is 8.14, so we want last from 8.13 and last form 8.12. Is that right? I can edit the requirements in this PR too.

---

Matched the version requirements in https://github.com/elastic/elastic-agent/pull/4535/commits/5c68484d56228f9991fd55812a6980a62a48fe54

---

Perfect thanks!

Most patch releases these days don't even contain changes to agent, and the changes that do exist are minor bug fixes, so there's no value in additional testing within the same minor usually.